www/class/centreonLDAP.class.php

```diff
@@ -1151,7 +1153,7 @@ public function setGeneralOptions($arId, $options)
             );
         }
         $knownParameters = $this->getLdapParameters();
-        if (isset($options["bind_pass"]) && $options["bind_pass"] === CentreonAuth::PWS_OCCULTATION) {
+        if (isset($options["bind_pass"]) && $options["bind_pass"] === CentreonAuth::PWS_OCCULTATION && $isUpdate) {
```

Comment:
are you sure that $isUpdate is required here and don't introduce a different behavior ?